src-plugins/itkDataImage/readers/itkDCMTKDataImageReader.cpp

```diff
@@ -396,7 +396,15 @@ bool itkDCMTKDataImageReader::readInformation (const QStringList& paths)
         //Dimensions
         //NumberOfDimensions
         medData->setMetaData(medMetaDataKeys::Orientation.key(),     d->io->GetOrientation().c_str());
-        //Origin
+
+        // Add Origin
+        QString origin = "";
+        for (int i=0; i<d->io->GetNumberOfDimensions(); ++i)
+        {
+            origin += QString::number(d->io->GetOrigin(i)) + QString(" ");
+        }
+        medData->setMetaData(medMetaDataKeys::Origin.key(), origin.trimmed());
```

Comment:
The problem is the origin metadata is not stored in the MUSIC database. This metadata will get lost as soon as the data is imported and reloaded.

Reply:
```medDataManager::getMetaData``` calls ```medDatabaseController::metaData``` which queries the SQL tables. If the requested column doesn't exist in the table (i.e. Origin, FlipAngle etc.) it won't return anything.

Reply:
You can use DB Browser for SQLite (https://sqlitebrowser.org) to access the SQL database directly (just open the "db" file) and see what is stored, test changes etc.

Reply:
Thanks for the comments. I'm not sure how to handle that. The echo time/flip angles/repetition time values was intended to be extracted from the DICOM, as in https://github.com/Inria-Asclepios/medInria-public/pull/432/files but indeed i knew it would disappear if you export it as .mha for instance, and import it again. Echo time/Etc should not change, but origin and size can change because of a process. 
Is it maybe possible that in `mscDatabaseMetadataItemDialog.cpp` i get back `medMetaDataKeys` as in `medDatabaseView::onMetadataRequested` and that i manually add origin/size/direction from the itk data? Hum

Reply:
You don't even need to export to have the problem. When the data is imported it is saved as .mha inside MUSIC, so when viewing the EchoTime etc. values using the QDialog added in https://github.com/Inria-Asclepios/medInria-public/pull/432, you will get empty values unless you do this immediately after import OR you only import in the temporary database (because in that case the medAbstractData stays in memory). There is no workaround to avoid changing the SQL tables.

Reply:
(If you manually add the data in ```mscDatabaseMetadataItemDialog``` it will load the data in memory every time someone wants to just display metadata which is not ideal.

Reply:
This shouldn't be too hard to do. You could search for how a metadata like "orientation" is handled and do the same for the missing ones. The only other thing would be to add the columns in the series table. A google search for "add SQLite column if not exist" should give a solution, and this can be added somewhere in the database management classes.

Reply:
In order to be sure about the bug: 

i opened a data in Reslice -> Change the dimension to 500x500x50 -> Save. The temporary data metadata shows 500x500x50 even after 5 sec. I definitively same them -> even after 5 seconds it's ok.

I tried with Reslice moving the center of the volume. The origin of this volume is displayed in metadata as -34,-216,-252 even after 5 sec. Definitive saving: origin disappears.

I'm looking at the problem.